docs/pages/build-reference/variants.mdx

```diff
@@ -79,13 +79,31 @@ export default {
   },
   android: {
     package: IS_DEV ? 'com.myapp.dev' : 'com.myapp',
-  },
+  }
 };
 ```
 
 In the above example, the environment variable `IS_DEV` is used to differentiate between the development and production environment. Based on its value, the different Application IDs or Bundle Identifiers are set for each variant.
 
-> **Note**: If you are using any libraries that require you to register your application identifier with an external service to use the SDK, such as Google Maps or Firebase Cloud Messaging (FCM), you'll need to have a separate configuration for that API for the `android.package` and `ios.bundleIdentifier`. You can also swap this configuration using the same approach as above.
+#### Additional app variant customizations
+
+You may wish to customize other aspects of your app on a per-variant basis. You can swap any configuration that you used previously in **app.json** using the same approach as above.
+
+Some examples:
```

Comment:
```suggestion
**Examples:**
```

Since we are already using H4 level of heading for this whole section, it might be good to highlight it here.